packages/components/message/src/message.ts

```diff
@@ -85,10 +85,16 @@ export interface MessageHandle {
 export type MessageParams = Partial<MessageOptions> | string | VNode
 export type MessageParamsTyped = Partial<MessageOptionsTyped> | string | VNode
 
-export type MessageFn = ((options?: MessageParams) => MessageHandle) & {
+export type MessageFn = ((
+  options?: MessageParams,
+  appContext?: null | AppContext
```

Comment:
Can we remove `null` here?
```suggestion
  appContext?: AppContext
```

Reply:
No, it is simply mapping for `Vue`'s typing

Reply:
I didn't get your point.

Reply:
<img width="1366" alt="image" src="https://user-images.githubusercontent.com/15975785/155449586-75532f08-e65e-45f9-ab67-6aeeafcfe86f.png">

Reply:
OK.